src/theme/index.js

```diff
@@ -0,0 +1,48 @@
+export const THEME_VARIANTS = {
+    DARK: 'dark',
+    LIGHT: 'light'
+};
+
+export const theme = {
```

Comment:
refactor this

Reply:
I think we need to style with theme couple another UI elements besides app background, so it will be easier to catch weaknesses in organization of code